substrate/frame/conviction-voting/src/tests.rs

```diff
@@ -59,9 +59,12 @@ impl pallet_balances::Config for Test {
 	type AccountStore = System;
 }
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Who;
```

Comment:
Why not make the type generic over `AccountId`? This can later be `T::AccountId`

Reply:
The "who" is not relevant for conviction voting so in this tests I just added a simple struct as placeholder, this struct is not used in the actual implementation of `Polling` in pallet referenda where it's set as the runtime origin. Perhaps I don't get your point on why `Who` would need to accept a generic field or is it in the `TestPollState` where you suggest the generic parameter? anyway let me know to clarify :)